pages/signUp-signIn.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../style/style.css" />
+    <title>Sign Up/In Page</title>
+  </head>
+  <body class="signup-page">
+    <header></header>
+    <section class="container">
+      <div class="SwitchBtn">
+        <button id="signInBtn">SIGN IN</button>
+        <button id="signUpBtn">SIGN UP</button>
+      </div>
+      <div class="LogoDiv">
+        <img src="../images/Logo.svg" alt="" />
+      </div>
+      <div id="signUpDiv" class="InputDiv">
+        <input id="uUserName" type="text" placeholder="User Name" />
+        <input id="uEmail" type="text" placeholder="Email" />
+        <input id="uPassword" type="text" placeholder="Password" />
+        <input id="uConfPassowrd" type="text" placeholder="Confirm Password" />
+        <button>Sign up</button>
+      </div>
+      <div id="signInDiv" class="InputDiv">
+        <input id="Email" type="text" placeholder="Email" />
+        <input id="Password" type="text" placeholder="Password" />
```

Comment:
I am thinking of using a password type field because when a user types in the password will appear in dots  (more safety)  and not the characters of the password which user enters